### PR TITLE
Email confirmation and verification

### LINKED ISSUE #45: Registration confirmation page
We need a dedicated page to put in the email for registration confirmation. It will be called like

`confirm?token=<lalala>`

See AuthenticationController for the type of requests needed. Fairly straightforward

### PR DESCRIPTION
**Pull request by martijn9612**
Fixes #45

### CODE REVIEW

**Comment by skamoen** [comment]
When opening the homepage, I'm getting `GET http://localhost:5100/api/v1/confirmRegistration?token= 404 (Not Found)` in the console

File: app/elements/lancie-login/lancie-register-confirmation.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      properties: {
+        token: {
+          type: String,
+          value: ''
+        }
+      },
+      
+      ready: function() {
+        this.token = window.location.href.split('token=')[1];
+        this.$.ajaxConfirmRegistration.generateRequest();
```
Comment (TimvdLippe) [inline app/elements/lancie-login/lancie-register-confirmation.html]: I think this should be in `attached` instead of `ready`.

**Comment by skamoen** [comment]
The mail states `https://areafiftylan.nl/confirmRegistration?token=8aab1824-8805-4fbb-940b-a5b9ea376f10`. When going to localhost:5100/confirmRegistration?token=8aab1824-8805-4fbb-940b-a5b9ea376f10, it fails due to the missing hasbang

**Comment by skamoen** [comment]
When going to the right URL, you're instantly redirected to the login screen. A message stating that the verification is successful would be good. This message could contain a link to My Area or Tickets

File: app/elements/lancie-login/lancie-register-confirmation.html (hunk 1 above)
Comment (martijn9612) [inline app/elements/lancie-login/lancie-register-confirmation.html]: I didn't know that this was a thing, thanks!

File: app/elements/routing.html [hunk 2]
```diff
@@ -26,19 +26,47 @@
...
   page('/orderfailure', function() {
-    app.route = 'order-failure'
+    app.route = 'order-failure';
   });
 
+  page('*', function() {
+    app.route = '404';
+  });
+
+  // page('*', function() {
```
Comment (TimvdLippe) [inline app/elements/routing.html]: Please remove commented code

**Comment by TimvdLippe** [comment]
One small nit, other than that LGTM

**Comment by TimvdLippe** [comment]
Appears to be fixed in #63

**Pull request closed without merging**